source/dbo/Tables/Data Migration/MigrationRunResults.sql

```diff
@@ -11,13 +11,15 @@
 	[SourceSystem] [nvarchar](10)  NULL,
 	[LegacyHospitalId] [varchar](50)  NULL,
 	[LegacyHospitalName] [nvarchar](255)  NULL,
-	[EtsTreatmentOutcome] [varchar] (30) NULL,
+	[LegacyOutcomeId] [int] NULL,
+	[EtsTreatmentOutcome] [varchar] (50) NULL,
```

Comment:
It may not be worth it, but do you think we should rename this field `LegacyTreatmentOutcome`?

Reply:
We should, but it's too much hassle at this stage given it then affected the report as well.  I decided not to make things worse by introducing a field called 'EtsOutcomeId' when it patently wasn't, but this report is nearly at the end of its natural life now anyway.